src/cluster_asm.c

```diff
@@ -240,6 +241,8 @@ asmTask *asmTaskCreate(void) {
     task->create_time = server.mstime;
     task->start_time = 0;
     task->done_time = 0;
+    task->dest_offset = 0;
+    task->source_offset = 0;
```

Comment:
putting them in `asmTaskReset` is enough

Reply:
interesting, they are in asmTaskReset() but still linter shows a silly warning.. It confused me.